### PR TITLE
Added functional --all-branches option.

### PR DESCRIPTION
**Pull request by evangineer**
Include all branches on repos with this option.

Satisfies #62

### CODE REVIEW

File: bin/repoxplorer-github-organization [hunk 1]
```diff
@@ -97,8 +100,21 @@ if __name__ == "__main__":
             if r.name not in top:
                 continue
         data = {r.name: {"template": args.org}}
-        if r.default_branch != 'master':
```
Comment (morucci) [inline bin/repoxplorer-github-organization:100]: Why not only:
```
branches = set(r.default_branch)
if args.all_branches:
  for branch in r.branches():
    branches.add(branch)
branches = list(branch)

Comment (evangineer) [reply bin/repoxplorer-github-organization:100]: Don't you mean?

`branches = list(branches)`

Comment (evangineer) [reply bin/repoxplorer-github-organization:100]: Have simplified the code in line with your suggestion.

I'm still seeing issues though.

**Comment by morucci** [comment]
Hi, I've tested you code. The yaml generated file looks good to me. I think it can be merged but first
could you explain a bit more what are you seeing as issues ? Could you give me a way to reproduce the issue ?

**Comment by evangineer** [comment]
Focusing on something else right now, but I suspect the problem is with the indexer not properly handling the multiple branches.

**Comment by morucci** [comment]
I've merged your changes. See https://softwarefactory-project.io/r/19761. This repository is just a mirror of the official repo on softwarefactory-project.io. https://github.com/morucci/repoxplorer/commit/be40c7aba7b0b15a200cbc8c5c3ac23cc06ba779

For the issue you mentioned, I'll have a deeper look. That's weird because this is tested in the tests suite here https://github.com/morucci/repoxplorer/blob/master/repoxplorer/tests/test_indexer_git.py#L584

**Comment by morucci** [comment]
Thanks !

**Pull request closed without merging**